src/winforms/toga_winforms/widgets/slider.py

```diff
@@ -39,12 +39,22 @@ def winforms_scroll(self, sender, event):
             self.interface.on_change(self.interface)
 
     def winforms_mouse_down(self, sender, event):
+        """
+        Since picking and releasing the slider is also a change, calling the
+            on_change method.
+        """
         if self.container and self.interface.on_press:
             self.interface.on_press(self.interface)
+        self.winforms_scroll(sender, event)
 
     def winforms_mouse_up(self, sender, event):
+        """
+        Since picking and releasing the slider is also a change, calling the
+            on_change method.
+        """
         if self.container and self.interface.on_release:
             self.interface.on_release(self.interface)
+        self.winforms_scroll(sender, event)
```

Comment:
I think this should go *before* the `on_release` handling... effectively, the sequence of events is "down, scroll, scroll, scroll, release".

Reply:
Completely agree. Entered the new logic.